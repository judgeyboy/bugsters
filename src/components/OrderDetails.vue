<template>
  <div>
    <h2>Order Details</h2>
  </div>
</template>

<script>
export default {
  name: 'OrderDetails'
}
</script>
