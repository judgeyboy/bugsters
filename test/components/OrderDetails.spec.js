import {shallow} from 'vue-test-utils'

import OrderDetails from '@/components/OrderDetails'

describe('OrderDetails component', () => {
  it('should be a vue instance', () => {
    const wrapper = shallow(OrderDetails)
    expect(wrapper.isVueInstance()).toBe(true)
  })

  it('should match snapshot', () => {
    const wrapper = shallow(OrderDetails)
    const $html = wrapper.vm.$el.outerHTML
    expect($html).toMatchSnapshot()
  })
})
