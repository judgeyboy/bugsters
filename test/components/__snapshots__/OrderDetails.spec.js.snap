// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrderDetails component should match snapshot 1`] = `

<div>
  <h2>
    Order Details
  </h2>
</div>

`;
